Makefile

```diff
@@ -0,0 +1,18 @@
+.PHONY: runserver migrate populatedb test help
+
+help: ## Display this help message
+	@echo "Saleor Development Commands:"
+	@echo
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+runserver: ## Start the development server with auto-reload
+	uvicorn saleor.asgi:application --reload
+
+migrate: ## Run database migrations
+	python manage.py migrate
+
+populatedb: ## Populate database with example data and create superuser
+	python manage.py populatedb --createsuperuser
+
+test: ## Run tests with database reuse
+	pytest --reuse-db
```

Comment:
you can also add scripts from package json? its build-schema & release. we will have the in one place

Reply:
Done in [dd1a6f7](https://github.com/saleor/saleor/pull/17351/commits/dd1a6f7911a6735a305fc453eb1745965718b6ff)